client/src/app/shared/components/logo/logo.component.ts

```diff
@@ -0,0 +1,79 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { MediaManageService } from '../../../site/mediafiles/services/media-manage.service';
+
+export enum possibleLogoActions {}
+
+/**
+ * Reusable Logo component for Apps.
+ *
+ * Following actions are possible:
+ * * "logo_projector_main"
+ * * "logo_projector_header"
+ * * "logo_web_header"
+ * * "logo_pdf_header_L"
+ * * "logo_pdf_header_R"
+ * * "logo_pdf_footer_L"
+ * * "logo_pdf_footer_R"
+ * * "logo_pdf_ballot_paper"
+ *
+ * ## Examples:
+ *
+ * ### Usage of the selector:
+ *
+ * ```html
+ * <os-logo
+ *   inputAction="logo_projector_main">
+ * </os-logo>
+ * ```
+ */
+@Component({
+    selector: 'os-logo',
+    templateUrl: './logo.component.html',
+    styleUrls: ['./logo.component.scss']
+})
+export class LogoComponent implements OnInit {
+    /**
+     * Holds the actions for logos. Updated via an observable
+     */
+    public logoActions: string[];
+
+    /**
+     * decides based on the actionString how to display the logo
+     */
+    @Input()
+    public inputAction: string;
+
+    /**
+     * The consotructor
+     * @param mmservice The Media Manage Service
+     */
+    public constructor(private mmservice: MediaManageService) {}
+
+    /**
+     * Initialization function
+     */
+    public ngOnInit(): void {
+        this.mmservice.getLogoActions().subscribe(action => {
+            this.logoActions = action;
+        });
+    }
+
+    /**
+     * gets the image based on the inputAction.
+     * Possible inputActions are in the class description.
+     */
+    public getImage(): string {
+        let path = '';
+        if (this.mmservice !== undefined) {
+            if (MediaManageService.isImageConfigObject(this.mmservice.getMediaConfig(this.inputAction))) {
+                const imageConfig = this.mmservice.getMediaConfig(this.inputAction);
+                path = imageConfig.path;
+            }
+        }
+        if (path === '') {
+            // TODO: Replace with config.default
+            path = '/assets/img/openslides-logo-h.svg';
```

Comment:
@FinnStutzenstein 
do you know if the default logo is written in a config or a constant?